src/renderer/base/renderer.cpp

```diff
@@ -330,13 +346,20 @@ bool Renderer::_CheckViewportAndScroll()
     coordDelta.X = srOldViewport.Left - srNewViewport.Left;
     coordDelta.Y = srOldViewport.Top - srNewViewport.Top;
 
-    std::for_each(_rgpEngines.begin(), _rgpEngines.end(), [&](IRenderEngine* const pEngine) {
-        LOG_IF_FAILED(pEngine->UpdateViewport(srNewViewport));
-        LOG_IF_FAILED(pEngine->InvalidateScroll(&coordDelta));
-    });
-    _srViewportPrevious = srNewViewport;
+    if (coordDelta.X != 0 || coordDelta.Y != 0)
+    {
+        std::for_each(_rgpEngines.begin(), _rgpEngines.end(), [&](IRenderEngine* const pEngine) {
+            LOG_IF_FAILED(pEngine->UpdateViewport(srNewViewport));
+            LOG_IF_FAILED(pEngine->InvalidateScroll(&coordDelta));
+        });
+        _srViewportPrevious = srNewViewport;

```

Comment:
I bet this is it

Reply:
I changed this and pushed. Is it still happening?